demos/demo-minimal-js/index.html

```diff
@@ -224,6 +224,12 @@
           type="button"
           value="resize widget"
         />
+        <input
```

Comment:
We could move this button to "Other events" but it would change the demo widget size. Any thoughts?